index.html

```diff
@@ -1149,12 +1151,54 @@ <h3>Types</h3>
         </p>
       </section>
 
+
+
+      <section>
+        <h3>Credential Subject</h3>
+
+        <p>
+A <a>verifiable credential</a> contains <a>claims</a> about a <a>subject</a>.
+The <a>subject</a> of the credential MUST be expressed using the
+<code>credentialSubject</code> <a>property</a>.
+        </p>
+
+        <dl>
+          <dt>credentialSubject</dt>
+          <dd>
+The value of the <code>credentialSubject</code> <a>property</a> MUST be
+an object that contains one or more properties that are related to the
+<a>subject</a> of the <a>verifiable credential</a>. The object MAY contain an
+<code>id</code> as described in Section <a href="#identifiers"></a>.
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of the credentialSubject property">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  <span class="highlight">"credentialSubject"</span>: {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science in Mechanical Engineering"
+    }
+  },
+  "proof": { ... }
+}
+        </pre>
+
+      </section>
+
       <section>
         <h3>Issuer</h3>
 
         <p>
-<a>Issuer</a> information can be expressed using the following
-<a>properties</a>:
+An identifier for the <a>issuer</a> of a <a>verifiable credential</a> MUST be
```

Comment:
Same comment, separation of concerns, impossibility of testing, etc.

A verifiable credential MUST contain an issuer property. The value of the issuer property MUST be a URI. It is RECOMMENDED that dereferencing . . .